gtars/src/overlap/ailist.rs

```diff
@@ -156,25 +221,22 @@ where
         starts: &[I],
         ends: &[I],
         max_ends: &[I],
-        data_list: &[T],
+        data_list: &[Interval<I, T>],
     ) -> Vec<Interval<I, T>> {
         let mut results_list = Vec::new();
         let mut i = starts.partition_point(|&x| x < end);
 
         while i > 0 {
             i -= 1;
-            if start > ends[i] {
+            // maintain start inclusive, end exclusive
+            if start >= ends[i] {
```

Comment:
Note - this changes the behavior to be the same as my rust bits impl, with the half open range.

Reply:
Got it -- I think its good to be consistent so this is fine